### DISCUSSION TITLE
System-Design/Load-Balancers/Load-Balancer

### DISCUSSION THREAD

**Discussion by giscus** [xy-241/CS-Notes#6] [General]
# System-Design/Load-Balancers/Load-Balancer

Abstract § A Reverse Proxy (反向代理) that does Load Balancing 2 types Application Load Balancer and Network Load Balancer. The example above is an External Application Load Balancer Benefits § Failover Capability § If one server goes offline, all the traffic will be routed to other servers by Load Balancer.

https://notes.yxy.ninja/System-Design/Load-Balancers/Load-Balancer

**Comment by OliverTeo288** [comment]
What are the popular routing types for load balancers? Eg Weight-based routings

**Reply by xy-241** [reply]
<img src="https://c.tenor.com/QEviHhIPURYAAAAC/tenor.gif" width="200">

The two most common ones are probably Round Robin and Weight Round Robin. I have included those two into the note. Let me know if there are any other interesting or useful load balancing strategies!
